src/pages/about-koselugo.astro

```diff
@@ -1,9 +1,82 @@
 ---
+import Section from "../components/Section.astro";
 import Layout from "../layouts/Layout.astro";
 import SimpleHero from "../components/SimpleHero.astro";
+import OneSource from "../components/OneSource.astro";
 ---
 
 <Layout title="About Koselugo">
-    <SimpleHero folder="about-koselugo" alt="" />
-    <div class="wrapper"></div>
-</Layout>
+    <div class="wrapper about-koselugo">
+        <SimpleHero folder="about-koselugo" alt="" />
+        <Section>
+          <Fragment slot="section-title">
+            WHAT IS <span class="color-yellow">KOSELUGO?</span>
+          </Fragment>
+          <div class="wrapper about-koselugo__what-is">
```

Comment:
Now Section component includes the wrapper. Please remove it from Section content

Reply:
Do I just have to remove the wrapper class? not the element, right?

Reply:
Is there a way to give a custom className to the Web Component?